### PR TITLE
Update songivan00 docs

### PR DESCRIPTION
**Pull request by songivan00**
Updated docs for v1.1

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -7,17 +7,17 @@ We are a team based in the [School of Computing, National University of Singapor
...
 You can reach us at the email `seer[at]comp.nus.edu.sg`
 
-## Project team
+## CodeConnect
 
-### John Doe
+### Song Ivan
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/Songivan.png" width="200px">
```
Comment (angkl0) [inline docs/AboutUs.md:14]: The filename of the profile photo should be docs/images/github_username_in_lower_case.png
Note the need for lower case
https://nus-cs2103-ay2223s1.github.io/website/schedule/week7/project.html

**Review by angkl0 — CHANGES_REQUESTED** [review]

File: docs/AboutUs.md (hunk 1 above)
Comment (songivan00) [reply docs/AboutUs.md]: okay will change thank you

**Review by angkl0 — APPROVED** [review]
Looks good!

**Review by angkl0 — DISMISSED** [review]

File: docs/team/songivan00.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* **Documentation**:
+  * User Guide:
+    * Added documentation for the features `list` and `listc`
+  * Developer Guide:
+    * Added Non-Functional Requirements
+
+* **Community**: to be added soon
+
+* **Tools**: to be added soon
+* _{you can add/remove categories in the list above}_
```
Comment (danzzzerl) [inline docs/team/songivan00.md:29]:
```suggestion
* _{you can add/remove categories in the list above}_

```

**Review by danzzzerl — APPROVED** [review]
added newline for songivan00.md file

**Review by angkl0 — APPROVED** [review]

**Pull request merged**